pkg/yamlmeta/type.go

```diff
@@ -6,6 +6,11 @@ package yamlmeta
 import (
 	"fmt"
 	"github.com/k14s/ytt/pkg/filepos"
+	"github.com/k14s/ytt/pkg/structmeta"
+)
+
+const (
+	AnnotationSchemaNullable structmeta.AnnotationName = "schema/nullable"
```

Comment:
Feels like this is a schema annotation. Shouldn't this be in the schema package?